### ISSUE TITLE
Log Explorer not working?

### ISSUE THREAD

**Issue by smalex-z** [SasView/sasview#2539]
**Describe the bug**
The Log Explorer in Sasview fails to display any messages, from Information such as computation progress to warning. Messages show up in the terminal but fail to show up in the Log Explorer.  @butlerpd suggests it may have to do with level priority of messages from the terminal - apparently they had a similar issue at a code camp recently.

**To Reproduce**

Computation Progress
1. Load a file into the Generic Scattering Calculator
2. Debye full Average (Preferably with a higher # of Bins)
3. Look at Terminal vs Log Explorer- Terminal will state Computation started and give you percentage updates every 20% or so, but Log Explorer will remain empty

Warnings
1. Load a file into the Generic Scattering Calculator that is not supported (ie. .MD)
2. Look at Terminal vs Log Explorer- Terminal will state "root - WARNING - The selected file does not have a suitable file extension" while Log Explorer remains empty.

**Expected behavior**
The Log Explorer should detail any info messages with high enough priority and all warnings (according to @butlerpd)

**Screenshots**

**SasView version (please complete the following information):**
 - Version: Main Branch

**Operating system (please complete the following information):**
 - OS: WIndows 11

**Comment by lucas-wilkins** [comment]
Probably need to look at the filters in `system/log.ini`

**Comment by krzywon** [comment]
@smalex-z, @butlerpd, and I looked closer and the logger emit was commented out in the PySide merge PR (#2478, [SasviewLogger.py#L21](https://github.com/SasView/sasview/blob/main/src/sas/qtgui/Utilities/SasviewLogger.py#L21)). It might be a similar issue to https://stackoverflow.com/questions/16538654/pyside-qt-too-many-arguments-to-connect-a-signal-to-a-slot, where the stackoverflow user is overriding their `connect` method, but we are overriding `emit` in our QtHandler, which coincidentally takes 2 arguments.

**Comment by smalex-z** [comment]
Looks like the Logging package within anaconda3>Lib>Logging is hijacking the emit signal.

**Comment by pkienzle** [comment]
There is a name collision between "emit" in QtHandler and "emit" in logging.Handler. You can avoid this separating the QtSignal from the logging handler:
```python
import os
import sys
import logging

from PySide6.QtCore import QObject, Signal

class QtPostman(QObject):
    """
    Version of logging handler "emitting" the message to custom stdout()
    """
    messageWritten = Signal(str)

    def __init__(self):
        QObject.__init__(self)

    def forward(self, message):
        self.messageWritten.emit('%s\n'%message)

class QtLogger(logging.Handler):
    def __init__(self):
        logging.Handler.__init__(self)
        self.postman = QtPostman()

    def emit(self, record):
        message = self.format(record)
        if message:
            self.postman.forward(message)

def setup_qt_logging():
    # Define the default logger
    logger = logging.getLogger()

    # Add the qt-signal logger
    handler = QtLogger()
    handler.setFormatter(logging.Formatter(
        fmt="%(asctime)s - %(levelname)s: %(message)s",
        datefmt="%H:%M:%S"
    ))
    logger.addHandler(handler)

    return handler
```
and using updating the signal to use the qt postman:
```python
        # Fork off logging messages to the Log Window
        handler = setup_qt_logging()
        handler.postman.messageWritten.connect(self.appendLog)
```

**Comment by pkienzle** [comment]
You don't seem to need the extra `'\n'`. The following is slightly more compact:
```python
class QtPostman(QObject):
    messageWritten = Signal(str)

class QtHandler(logging.Handler):
    """
    Emit python log messages through a Qt signal. Receivers can connect
    to *handler.postman.messageWritten* with a method accepting the
    formatted log entry produced by the logger.
    """
    def __init__(self):
        logging.Handler.__init__(self)
        self.postman = QtPostman()

    def emit(self, record):
        message = self.format(record)
        if message:
            self.postman.writtenMessage.emit(message)
```

**Comment by smalex-z** [comment]
Closed in #2545
